src/db/controller/client.ts

```
@@ -51,15 +51,35 @@
     if (header.signedHeader?.header?.time.seconds) {
-      client.last_update_time = Number(header.signedHeader.header.time.seconds)
+      const lastUpdateTime = Number(header.signedHeader.header.time.seconds)
+      if (lastUpdateTime > client.last_update_time) {
+        client.last_update_time = lastUpdateTime
+      }
     }
 
-    insert(DB, this.tableName, client)
+    update<ClientTable>(
+      DB,
+      this.tableName,
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)